src/components/Timer.js

```diff
@@ -0,0 +1,19 @@
+class Timer {
+  constructor() {
+    this.ref = document.querySelector(".clock");
+  }
+  currentTime() {
+    let date = new Date();
+
+    return date.toLocaleTimeString();
+  }
+
+  render(props) {
+    this.ref.innerHTML = `
+    <span>${this.currentTime()}</span>
+    `;
+    setTimeout(() => this.render(props), 1000);
+  }
```

Comment:
```suggestion

  render() {
    this.ref.innerHTML = `
    <span>${new Date().toLocaleTimeString()}</span>
    `;
    setTimeout(() => this.render(), 1000);
  }
```
`props` was not used so, we can skip it it
`currentTime` could be onelined

Reply:
code edited.